mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -203,6 +205,18 @@ func detachPrimaryBlockDevice(c cluster.TestCluster, m platform.Machine) {
 		if err := m.(platform.QEMUMachine).RemovePrimaryBlockDevice(); err != nil {
 			c.Fatalf("failed to delete the first boot disk: %v", err)
 		}
+		// Check if we can still SSH into the machine. We've noticed sometimes
+		// that after removing the primary device, we lose connectivity.
+		if err := ut.Retry(5, 3*time.Second, func() error {
```

Comment:
@dustymabe This is another occurrence we'll want to switch over to your new `Retry` function.